src/Agent.Worker/Container/ContainerInfo.cs

```diff
@@ -68,6 +125,32 @@ public List<MountVolume> MountVolumes
             }
         }
 
+        public IDictionary<string, string> UserPortMappings
```

Comment:
i don't expect this to be public. :D

Reply:
Like UserMountVolumes, it could be made internal so that it can be traced in ContainerOperationProvider. Or, it could just be private, and not traced. Tracing may not be that useful since its only tracing exactly what the user has in their yaml anyways, so its not secret or internal information that the user would not know for debugging. 

However, tracing may be useful because we can trace out the expanded values. To minimize visibility, I think `internal` is the best